src/selectors/temperatureSync.js

```diff
@@ -47,11 +47,13 @@ export const selectTemperatureSyncIsComplete = ({ temperatureSync }) => {
 };
 
 export const selectTemperatureSyncStateMessage = ({ temperatureSync }) => {
-  const { syncState, syncError } = temperatureSync;
+  const { syncState, syncError, lastTemperatureSync } = temperatureSync;
+  const formattedDate = moment(lastTemperatureSync).format('D.M.YYYY');
```

Comment:
Any reason for `D.M.YYYY` (as opposed to, e.g. `DD.MM.YYYY`)?

Reply:
This is whats used for normal sync, and `.getDate()` returns for example, `9`, for today (So trying to match existing stuffs)